.github/scripts/logs.py

```diff
@@ -0,0 +1,24 @@
+import boto3
```

Comment:
Could this be named something a bit more descriptive, `logs.py` is a bit generic?

Reply:
It's not as easy as that because the call to run this script is in tdr-github-actions and changing it will break existing jobs if we're not careful. There should be a proper versioning process for tdr-github-actions which I didn't do originally because I thought we didn't need it but now we do. I'll put a ticket in Jira.

Reply:
Ok thanks